test/fixtures/loopback_acl_help.txt

```diff
@@ -0,0 +1,17 @@
+Usage:
+ slc loopback:acl [options] 
```

Comment:
Can we move these files to a subdirectory, so that `test/fixtures` is not overcrowded? E.g. `test/fixtures/help-texts` or perhaps even `test/expected-help-texts`.